sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -561,8 +561,11 @@ class ExpressionParserSuite extends PlanTest {
       Literal(CalendarInterval.fromSingleUnitString(u, s))
     }
 
-    // Empty interval statement
-    intercept("interval", "at least one time unit should be given for interval literal")
```

Comment:
we shall still check the empty interval statement, now it shall produce a different error message?

Reply:
yea, antlr just throws an exception when hitting this case;
```
scala> sql("select cast('2018-01-12' as DATE) + 1 days").show
+---------------------------------------------------------------------------+
|CAST(CAST(CAST(2018-01-12 AS DATE) AS TIMESTAMP) + interval 1 days AS DATE)|
+---------------------------------------------------------------------------+
|                                                                 2018-01-13|
+---------------------------------------------------------------------------+

scala> sql("select cast('2018-01-12' as DATE) + interval").show
org.apache.spark.sql.AnalysisException: cannot resolve '`interval`' given input columns: []; line 1 pos 36;
'Project [unresolvedalias((cast(2018-01-12 as date) + 'interval), None)]
+- OneRowRelation
```